feat(aps): add monthly commitment (#4601)
scaleway/docs-content: pages/apple-silicon/quickstart.mdx

        </Message>
    - Enter a **Name** for your Mac mini, or leave the randomly-generated name in place.
    - Verify the **Estimated cost** for your Mac mini based on your chosen specifications.
4. Choose the commitment terms for your Mac mini. Two options are available:

>     - No commitment: You can terminate the subscription of your Mac mini at any time after the initial 24 hours, which is a mandatory period required by Apple's software license. [Learn more about Apple's software license requirements (Section 3.A)](https://www.apple.com/legal/sla/docs/macOSSequoia.pdf).

Reviewer: ```suggestion
    - No commitment: You can terminate the subscription of your Mac mini at any time after the initial 24 hours, a mandatory period required by Apple's software license. [Learn more about Apple's software license requirements (Section 3.A)](https://www.apple.com/legal/sla/docs/macOSSequoia.pdf).
```

>     - Commitment: You subscribe to a [Commitment Plan for Apple silicon](/apple-silicon/concepts/#commitment-plan) and benefit from a discounted monthly price depending on your commitment.

Reviewer: ```suggestion
    - Commitment: You subscribe to a [commitment plan for Apple silicon](/apple-silicon/concepts/#commitment-plan) and benefit from a discounted monthly price based on your choice.
```

    <Message type="important">

>       It is currently not possible to switch back from a monthly commitment to hourly billing.

Reviewer: ```suggestion
      Currently, it is not possible to switch back from a monthly commitment to hourly billing.
```
